src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/reference/inputs/EceWorkExperienceReferenceDeclaration.vue

```diff
@@ -2,6 +2,9 @@
   <v-row>
     <v-col cols="12" md="12" lg="12" xl="12">
       <v-row no-gutters>
+        <v-col cols="12" v-for="(systemMessage, index) in configStore.$state.systemMessages">
+          <Banner v-if="systemMessage.portalTags && systemMessage.portalTags.includes('REFERENCES')" type="info" :title="systemMessage.message" />
+        </v-col>
         <v-col>
```

Comment:
we have to import the Banner component here too. 

components: {Banner}, like this

![image](https://github.com/user-attachments/assets/985f8b8b-2fa8-486c-85cd-c24d5dd68c9d)